src/test/java/io/lenses/connect/smt/header/TimestampConverterTest.java

```diff
@@ -883,6 +886,66 @@ void testWithSchemaValuePrefixedFieldConversion_Seconds() {
     assertEquals(expectedDate.getTime(), header.value());
   }
 
+  @Test
+  void testWithRecordMetadataPrefixedFieldConversion_Seconds() {
+    Map<String, String> config = new HashMap<>();
+    config.put(TimestampConverter.TARGET_TYPE_CONFIG, "Timestamp");
+    config.put(TimestampConverter.FIELD_CONFIG, "_timestamp");
+    config.put(TimestampConverter.UNIX_PRECISION_CONFIG, "seconds");
+    config.put(TimestampConverter.HEADER_NAME_CONFIG, "ts_header");
+    // ts field is a unix timestamp with seconds precision
+    Schema structWithTimestampFieldSchema =
+        SchemaBuilder.struct().field("ts", Schema.INT64_SCHEMA).build();
+    Struct original = new Struct(structWithTimestampFieldSchema);
+    original.put("ts", DATE_PLUS_TIME_UNIX_SECONDS);
+
+    final TimestampConverter<SourceRecord> transformer = new TimestampConverter<>();
+    transformer.configure(config);
```

Comment:
I do see your point about TimestampConverter not actually creating any resources but IntelliJ isn't that smart.  Would it be worth either suppressing the warning or switch to using the try {} format to close the resources.
![Screenshot 2024-02-21 at 09 38 36](https://github.com/lensesio/kafka-connect-smt/assets/33483659/9e48b97a-d6cb-4fe0-bfa9-1cf3b5afabcd)